contrib/zmq/zmq_sub_py3.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+# Copyright (c) 2014-2016 The Bitcoin Core developers
+# Copyright (c) 2018 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import array
+import binascii
+import zmq
+import struct
+
+port = 28332
+
+zmqContext = zmq.Context()
+zmqSubSocket = zmqContext.socket(zmq.SUB)
+zmqSubSocket.setsockopt(zmq.SUBSCRIBE, b"hashblock")
+zmqSubSocket.setsockopt(zmq.SUBSCRIBE, b"hashtx")
+zmqSubSocket.setsockopt(zmq.SUBSCRIBE, b"rawblock")
+zmqSubSocket.setsockopt(zmq.SUBSCRIBE, b"rawtx")
+zmqSubSocket.connect("tcp://127.0.0.1:%i" % port)
+
+try:
+    while True:
+        msg = zmqSubSocket.recv_multipart()
+        topic = msg[0]
+        body = msg[1]
+        sequence = b"Unknown";
+        if len(msg[-1]) == 4:
+          msgSequence = struct.unpack('<I', msg[-1])[-1]
```

Comment:
This is nonstandard indentation and I am not sure how this will be interpreted. The same thing exists in the file that this is based on. I am not sure if this is a mistake, but my guess is that it is.